### PR TITLE
fix: default properties setup

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

**Comment by niekvanstaveren** [comment]
@miralemd In order for this to run in Nebula I think we need babel-polyfill to be added. Can you have a look as well?

File: src/object-properties.js [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+  /**
+   * @typedef {object}
+   */
+  style: {
+    /**
+     * @type {string}
+     */
+    label: '',
+  },
 };
```
Comment (miralemd) [inline src/object-properties.js]: I would suggest adding `actions: []` here to make sure it exists when the object is created

File: src/utils/style-formatter.js [hunk 2]
```diff
@@ -13,15 +13,12 @@ export default {
   getStyles(style, Theme) {
     // TODO: use constants for default values?
     let styles = 'width: 100%;height: 100%;font-weight: bold;';
-
-    if (style) {
+    if (Theme) {
```
Comment (haxxmaxx) [inline src/utils/style-formatter.js]: Should add a test for when Theme is undefined, given that could actually happen, otherwise remove the check.

Comment (haxxmaxx) [reply src/utils/style-formatter.js]: Also `Theme && (palette = Theme.getCurrent().properties.palettes.ui[0].colors)` might look nicer

Comment (Caele) [reply src/utils/style-formatter.js]: wrap the theme stuff in its own file as well, there is a chance that that palettes doesn't exist on the theme properties, so we need a way to fall back to something in that case. And for the case where Theme doesn't exist at all, like when outside of sense.

File: src/utils/theme-resolver.js [hunk 3]
```diff
@@ -12,5 +12,13 @@ const themeResolver = {
     }
     return 'none';
   },
+  getPalette: Theme => {
```
Comment (haxxmaxx) [inline src/utils/theme-resolver.js]: Nice!

**Pull request merged**